components/gfx/font_cache_thread.rs

```diff
@@ -216,7 +216,8 @@ impl FontCache {
                     url: url.clone(),
                     type_: RequestType::Font,
                     destination: Destination::Font,
-                    origin: url.clone(),
+                    // TODO: Add a proper origin - Can't import GlobalScope from gfx
```

Comment:
That would be fine for mocking in the tests, maybe. But in [font_cache_thread.rs](https://github.com/servo/servo/pull/16508/files/2a7262fa6ed76bbf5bb0c911fdbfb18693ce92db#diff-35c06da67ec4389f8f1f7350c3f960e8) we need the current origin so I can't just build it... maybe if I could access the current URL and then build an origin from it? I haven't figured out how to find the URL though.